test/bridge.test.ts

```diff
@@ -32,10 +32,11 @@ chai.use(solidity);
 const MAX_UINT256 = hre.ethers.constants.MaxInt256;
 
 // Amount of dai and usdc to transfer to the user. Issued twice for aDai and aUsdc
-const DAI_UNIT = 1000n * BigInt(10 ** 18);
-const USDC_UNIT = 1000n * BigInt(10 ** 6);
+const DAI_UNIT = BigInt(10 ** 18);
```

Comment:
nice, I've introduced these `1000n` when we used to have very small amounts of reward. good that it works even without it!